code/modules/projectiles/projectile/energy/ebow.dm

```diff
@@ -4,9 +4,18 @@
 	damage = 8
 	damage_type = TOX
 	nodamage = 0
-	knockdown = 100
-	stutter = 5
-	drowsy = 50
+	stamina = 60
```

Comment:
iirc stam is out of 100, so its a 2 hit stam crit.

Reply:
That is the ideal amount of shots, yes. Consistency is what I was aiming for with the weapon, and also why I have concerns about large energy crossbows. Again, thinking up ways to fix that. Shouldn't be a big issue for miniature energy crossbows.